src/Chapter1.hs

```diff
@@ -413,7 +414,7 @@ task is to specify the type of this function.
 -}
 
 squareSum x y = (x + y) * (x + y)
-
+squareSum :: Num a => a -> a -> a
```

Comment:
👍🏼 Nice!
We explain `Num` in the following chapters, so you are one step ahead 😸 

In here `Int -> Int -> Int` would also work, as a more specified type 🙂 

Also, in Haskell you usually write the type signature before the function body:

```haskell
foo :: TYPE SIGNATURE
foo = BODY
```